games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A free and open source chess database application"
+DESCRIPTION="With ChessX you can operate your collection of chess games \
+in many ways: browse, edit, add, organize, analyze, etc."
+HOMEPAGE="https://chessx.sourceforge.net/"
+COPYRIGHT="1998-2005 Gilles Vollant
+	1999-2004 Shane Hudson
+	2005 Ejner Borgbjerg
+	2005-2007 Marius Roets
+	2005-2007 Heinz R. Hopfgartner
+	2005-2010 Michal Rudolf
+	2005-2011 Sergey A. Tachenov
+	2007 Rico Zenklusen
+	2008 Aliaksandr Antonik
+	2010 P. Sereno
+	2011 Nokia Corporation and/or it's subsidiary(-ies)
+	2012-2016 Jens Nissen"
+LICENSE="GNU GPL v2
+	BSD (3-clause)"
```

Comment:
As of licenses:

- LGPL2, Zlib in `src/quazip`
- LGPL2.1 with Digia Qt LGPL Exception version 1.1 (you would have to dig this up yourself, sorry) in `src/gui/textedit.cpp`

Hopefully that's all